### ISSUE TITLE
The url in countRequest should not be hard coded

### ISSUE THREAD

**Issue by da-liii** [Algomancer/Full-Stack-Scala-Starter#9]

**Comment by Algomancer** [comment]
Agree, Feel free to send a PR. It is a hyper minimal example. We use code gen for request routes, but I excluded it due to becoming opinionated.

**Comment by hkirk** [comment]
I've created a version using autowire which are not hardcoding the Play server URL, but there are some other problems - could be a solutions
https://github.com/hkirk/Full-Stack-Scala-Starter
